content/get-started/quickstart/set-up-git.md

```diff
@@ -55,7 +55,7 @@ If you don't need to work with files locally, {% data variables.product.product_
 
 ## Next steps: Authenticating with {% data variables.product.prodname_dotcom %} from Git
```

Comment:
I suspect that we probably want to drop "Next steps" here, in the same way that we did in the other articles.

```suggestion
## Authenticating with {% data variables.product.prodname_dotcom %} from Git
```